ee/hogai/graph/root/nodes.py

```diff
@@ -229,9 +231,11 @@ def _run_and_format_insight(
             )
             return result
 
-        except Exception:
+        except Exception as err:
             # Skip insights that fail to run
-            capture_exception()
```

Comment:
Question: What's the logic behind:
- using `posthoganalytics.capture_exception` in some cases (like `rag/nodes.py`and `assistant.py`)
- and also while using `capture_exception` from `posthog/exceptions_capture.py`(that also imports `posthoganalytics.capture_exception`) in other cases (like `query_executor/nodes.py` or `root/nodes.py`)

As they use the same method underneath - it seems alright, but slightly confusing when to pick one or another.